src/data/api.ts

```diff
@@ -252,3 +253,34 @@ export async function fetchSubjectPage(
     layout: subjectPage.displayInTwoColumns ? 'double' : 'single',
   };
 }
+
+export async function search(
+  searchQuery: QueryToSearchArgs,
+  context: Context,
+): Promise<GQLSearch> {
+  const query = {
+    ...searchQuery,
+    'page-size': searchQuery.pageSize,
+    'context-types': searchQuery.contextTypes,
+    'resource-types': searchQuery.resourceTypes,
+    'language-filter': searchQuery.languageFilter,
+  };
+  const response = await fetch(
```

Comment:
We default to caching all GET calls, but it does'nt really make sense to cache search calls. You can pass a `'no-store'` cache option to disable caching.